src/Compatibility/Microsoft.DotNet.ApiSymbolExtensions/AssemblySymbolLoader.cs

```diff
@@ -16,6 +16,22 @@ namespace Microsoft.DotNet.ApiSymbolExtensions
     /// </summary>
     public class AssemblySymbolLoader : IAssemblySymbolLoader
     {
+        // This is a list of dangling .NET Framework internal assemblies that should never get loaded.
+        private static readonly HashSet<string> s_assembliesToIgnore = [
```

Comment:
How did you obtain this list?

Reply:
Eric gave me the list but the tool and the command that generated is `C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8>\tools\verifyClosure\verifyClosure.exe .\`